lib/SCSI2SD/src/firmware/mode.c

```diff
@@ -220,6 +220,23 @@ static const uint8_t ControlModePage[] =
 0x00, 0x00 // AEN holdoff period.
 };
 
+#ifdef ENABLE_AUDIO_OUTPUT
```

Comment:
We could consider adding some hook to `mode.c` to pass mode page requests to ZuluSCSI_cdrom.cpp. Something like `void scsiCDRomRequestModePage(uint8_t pageCode, int *pageFound, int *idx)` that would write to `scsiDev.data`. I would like to keep modifications to SCSI2SD files to minimum, to make it easier to merge changes back and forth when needed.

Reply:
That'd sounds fine. I mainly wanted to see if you had any intention of creating ZuluSCSI_mode.cpp with more comprehensive support for MODE SELECT functionality. I don't know how many devices even bother trying to set stuff that way so it may be more trouble than it's worth. As of now I'm not finding the current setup restrictive for getting basic playback working. Maybe this could be addressed in a separate PR to add additional playback features.

Reply:
Yeah, can be tidied up in the future if we need more modes.